docs/frameworks.md

```diff
@@ -58,27 +58,36 @@ For a full example, see Protractor’s own test: [/spec/mocha/lib_spec.js](/spec
 Using Cucumber
 --------------
 
-_Note: Limited support for Cucumber is available as of January 2015. Support for Cucumber in Protractor is maintained by the community, so bug fixes may be slow. For more information, see the [Cucumber GitHub site](https://github.com/cucumber/cucumber-js)._
+_Note: Cucumber is no longer included by default as of November 2015. You can integrate Cucumber with Protractor with the `custom` framework option. For more information, see the [Protractor Cucumber Framework site](https://github.com/mattfritz/protractor-cucumber-framework) or the [Cucumber GitHub site](https://github.com/cucumber/cucumber-js)._
 
 
 If you would  like to use the Cucumber test framework, download the dependencies with npm. Cucumber should be installed in the same place as Protractor - so if protractor was installed globally, install Cucumber with -g.
 
 ```
 npm install -g cucumber
+npm install --save-dev protractor-cucumber-framework
 ```
 
-Set the 'framework' property to cucumber, either by adding `framework: 'cucumber'` to the [config file](../spec/cucumberConf.js) or by adding `--framework=cucumber` to the command line.
+Set the 'framework' property to custom by adding `framework: 'custom'` and `frameworkPath: 'path/to/cucumberFramework'` to the [config file](../spec/cucumberConf.js)
 
 Options for Cucumber such as 'format' can be given in the config file with cucumberOpts:
 
 ```js
-cucumberOpts: {
-  format: "summary"
-}
-```
+var cucumberFrameworkPath = require('protractor-cucumber-framework').resolve();
 
-For a full example, see Protractor’s own test: [/spec/cucumber/lib.feature](/spec/cucumber/lib.feature).
+exports.config = {
+  // set to "custom" instead of cucumber.
+  framework: 'custom',
 
+  // path relative to the current config file
+  frameworkPath: cucumberFrameworkPath
```

Comment:
Why not just use `frameworkPath: 'protractor-cucumber-framework'`?  I'm pretty sure `frameworkPath` just gets passed directly to `resolve` anyway